### DISCUSSION TITLE
multiheads_finetuning data gives warning: using energy_key 'energy' is no longer safe

### DISCUSSION THREAD

**Discussion by AlghamdiNada** [ACEsuit/mace#764] [Q&A] [answered]
Hello,

When I use `multiheads_finetuning=true` I get warnings for energy, forces and stresses:

WARNING: Since ASE version 3.23.0b1, using energy_key 'energy' is no longer safe when communicating between MACE and ASE. We recommend using a different key, rewriting 'energy' to 'REF_energy'. You need to use --energy_key='REF_energy' to specify the chosen key name.

in my fine tuning data I have the keys as REF_energy and REF_forces.
It seems to me that the mp pt_head data has the keys as energy, forces and stresses which gives the warning. Can I safely ignore this warning? I don't get the warnings if use `multiheads_finetuning = false`

I am using mace version 0.3.9 and my config.yml file:
```
name: MACE
foundation_model: ~/.cache/mace/mace-medium-density-agnesi-stress.model #mace_agnesi_medium.model
loss: universal
default_dtype: float64
device: cuda
seed: 1
r_max: 4.5
restart_latest: true
pt_train_file: mp

train_file: ../0.xyz
valid_file: ../../1.xyz

multiheads_finetuning: True
num_samples: 12
E0s: '{3: -202.82202355577203, 9: -803.8441491412796}'
batch_size: 2
valid_batch_size: 10
max_num_epochs: 1000

compute_stress: true
energy_key: REF_energy
forces_key: REF_forces
stress_key: REF_stress_eV
##virials_key: None

energy_weight: 10
forces_weight: 100
stress_weight: 100
##virials_weight: 0

amsgrad: true
ema: true
ema_decay: 0.99
error_table: PerAtomMAE
scaling: rms_forces_scaling
swa: true
start_swa: 700
swa_energy_weight: 10
swa_forces_weight: 1
swa_stress_weight: 1
patience: 40
```

**Comment by ilyes319** [comment, marked as answer]
Hey @AlghamdiNada , indeed you can safely ignore this warning. It is just due to me hardcoding the MP head with these keys.
